lib/test.js

```diff
@@ -86,7 +86,7 @@ Test.prototype.plan = function (count) {
 
 	// in case the `planCount` doesn't match `assertCount,
 	// we need the stack of this function to throw with a useful stack
-	this.planStack = new Error().stack;
+	this.planStack = planStack || new Error().stack;
```

Comment:
Actually, I think it always is. I got it in my head that method might be called directly in our unit tests, but I'm pretty sure all our unit tests use `PublicApi` anyways.